src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomStoredFieldsFormat.java

```diff
@@ -98,7 +98,7 @@ public StoredFieldsWriter fieldsWriter(Directory directory, SegmentInfo si, IOCo
         return impl(mode).fieldsWriter(directory, si, context);
     }
 
-    StoredFieldsFormat impl(Lucene95CustomCodec.Mode mode) {
+    public StoredFieldsFormat impl(Lucene95CustomCodec.Mode mode) {
```

Comment:
Why this was changed to public? This is 100% internal impl

Reply:
got it, I'll refactor this back